packages/realm/src/Logger.ts

```diff
@@ -186,6 +186,33 @@ export type LogEntry = {
 export type LoggerCallback2 = (entry: LogEntry) => void;
 export type LoggerCallback = LoggerCallback1 | LoggerCallback2;
 
+/** @internal */
+export function toBindingLogger(logger: LoggerCallback) {
+  if (isLoggerWithLevel(logger)) {
+    return binding.Helpers.makeLogger((_, level, message) => {
+      logger(fromBindingLoggerLevelToLogLevel(level), message);
+    });
+  }
+  if (isLoggerWithOptions(logger)) {
```

Comment:
You know me ... but I wont die on that hill 😆

```suggestion
  } else if (isLoggerWithOptions(logger)) {
```

Reply:
Great, I'll take it 😉